src/main/java/software/aws/neptune/NeptuneDatabaseMetadata.java

```diff
@@ -45,7 +45,7 @@ public String getDatabaseProductVersion() throws SQLException {
 
     @Override
     public String getCatalogTerm() throws SQLException {
-        return "graph";
+        return "";
```

Comment:
Catalog will be still returned.
![image](https://user-images.githubusercontent.com/7035336/142365392-e400ecb3-a2e4-4a6b-9dd6-8fa187e7e0ef.png)

Need to fix `getCatalogs` method, for example this code should work - 
```    @Override
    public ResultSet getCatalogs() throws SQLException {
        // Note: return NO records to indicate no catalogs.
        return new EmptyResultSet(null);
    }
```

Reply:
Also need to alter Connection.getCatalog, Connection.setCatalog, and any of the DatabaseMetaData functions which return result sets which have catalog as a column (such as getTables, getSchemas, etc).

Reply:
I talked to Kyle about this.

Our Connection.getCatalog/setCatalog is okay.
Our DatabaseMetaData function getTables is okay.
Our DatabaseMetaData function **getSchemas needs to be fixed**.

In src/main/java/software/aws/neptune/common/gremlindatamodel/resultset/ResultSetGetSchemas.java we need the following updated (schema should be reported as null).
```
    static {
        CONVERSION_MAP.put("TABLE_SCHEM", "gremlin");
        CONVERSION_MAP.put("TABLE_CAT", null);
    }
```

To support this null return, we need to update software/aws/neptune/common/gremlindatamodel/resultset/ResultSetGetString.java
with:
`    private boolean wasNull = false;`
and 
```

    @Override
    protected Object getConvertedValue(final int columnIndex) throws SQLException {
        verifyOpen();
        final int index = getRowIndex();
        if ((index >= constantReturns.size()) || (index < 0)
                || ((columnIndex > columns.size()) || (columnIndex <= 0))) {
            throw SqlError.createSQLFeatureNotSupportedException(LOGGER);
        }
        final String key = columns.get(columnIndex - 1);
        if (constantReturns.get(index).containsKey(key)) {
            final Object value = constantReturns.get(index).get(key);;
            wasNull = value == null;
            return value;
        } else {
            throw SqlError.createSQLFeatureNotSupportedException(LOGGER);
        }
    }

    @Override
    public boolean wasNull() throws SQLException {
        return wasNull;
    }
```

Reply:
ResultSetGetCatalogs

    static {
        CONVERSION_MAP.put("TABLE_CAT", null);
    }

Reply:
public ResultSetGetCatalogs(final Statement statement) {
        super(statement, ImmutableList.of("TABLE_CAT"), 0, ImmutableList.of(CONVERSION_MAP));
    }